app/modules.go

```diff
@@ -175,6 +179,7 @@ func orderBeginBlockers(allModuleNames []string) []string {
 	// IBChost came after staking, before superfluid.
 	// TODO: Come back and delete this line after testing the base change.
 	ord.Sequence(stakingtypes.ModuleName, ibchost.ModuleName, superfluidtypes.ModuleName)
+	// We leave downtime-detector un-constrained.
```

Comment:
What do we mean by un-constrained here?

Reply:
It can go anywhere in the sequence of begin block runs